### PR TITLE
Biohackathon fixes

### PR DESCRIPTION
**Pull request by mvdbeek**
This PR collects all fixes done so far during the biohackathon and in preparation to test all tools on this repo. 

FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/obitools/macros.xml [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 `OBITools`_
 
-.. _OBITools: http://metabarcoding.org/obitools/
+.. _OBITools: https://pythonhosted.org/OBITools/welcome.html
```
Comment (nsoranzo) [inline tools/obitools/macros.xml]:
```suggestion
.. _OBITools: https://pythonhosted.org/OBITools/
```

File: data_managers/data_manager_rsync_g2/data_manager/data_manager_rsync.py [hunk 2]
```diff
@@ -75,9 +76,9 @@ def exec_before_job( app, inp_data, out_data, param_dict, tool=None, **kwd ):
     data_manager = app.data_managers.get_manager( tool.data_manager_id, None )
     data_table_entries = get_data_table_entries( param_dict )
     data_tables = load_data_tables_from_url( data_table_class=app.tool_data_tables.__class__ ).get( 'data_tables' )
-    for data_table_name, entries in data_table_entries.items():
+    for data_table_name, entries in list(data_table_entries.items()):
```
Comment (nsoranzo) [inline data_managers/data_manager_rsync_g2/data_manager/data_manager_rsync.py]: This should not be needed unless you add/remove keys from the dictionary.

Comment (mvdbeek) [reply data_managers/data_manager_rsync_g2/data_manager/data_manager_rsync.py]: agreed, this is a autopep8 leftover

**Comment by nsoranzo** [comment]
Not sure why kobas_annotate test does not produce output for 20', ping @blankclemens .

**Comment by mvdbeek** [comment]
It does eventually produce output when using github workflows, though it fails because of missing libbz2:
Screenshot 2019-11-27 at 17 16 00

**Comment by mvdbeek** [comment]
Anyway, probably better to do smaller PRs here to not get slowed down

**Comment by nsoranzo** [comment]
I've replicated the `kobas_annotate` issue on my laptop, it's stuck trying to create the conda environment:
```
conda create -y --quiet --override-channels --channel iuc --channel conda-forge --channel bioconda --channel defaults --name __kobas@2.1.1 kobas=2.1.1
```

After more than 20' it still hasn't solved the environment.

**Comment by bgruening** [comment]
@nsoranzo can we bump the conda recipe and get a new build?

**Comment by mvdbeek** [comment]
Yeah, it's been doing that all night on my laptop (I just added bzip2). The 2.1.1 recipe is not in the branch anymore, should we add that back ?

**Comment by nsoranzo** [comment]
@mvdbeek I'm going to push a fix in a bit, adding r-base 3.4 as additional requirement for kobas tools.

**Comment by mvdbeek** [comment]
Cool! If you want you could rebase out the mothur changes so that we actually get a chance at getting the tool tests to finish on time.

**Comment by nsoranzo** [comment]
> Cool! If you want you could rebase out the mothur changes so that we actually get a chance at getting the tool tests to finish on time.

Done!

**Pull request merged**